### ISSUE TITLE
Does this software have accounts function ？

### ISSUE THREAD

**Issue by sunny868** [loic-sharma/BaGet#591]
Does this software have an account function, which can be used by others to register and log in ? 
just like nuget.org sign in to push new package.

**Comment by patriksvensson** [comment]
@sunny868 No, only API key is supported afaik: https://loic-sharma.github.io/BaGet/configuration/#require-an-api-key

**Comment by sunny868** [comment]
Thanks
